using-mobile-extensions/adobe-analytics-mobile-services/README.md

```diff
@@ -9,29 +9,29 @@ As of **April 1, 2020**, Apple will no longer support UIWebView. To avoid any is
 {% hint style="info" %}
 The Adobe Analytics Mobile Marketing Add-on SKU is required to enable Mobile Services access to mobile acquisition, deep linking, geolocation, and mobile messaging capabilities.
 
-For more information, contact your Adobe CSM.
+For more information, please contact your Adobe Customer Success Manager.
 {% endhint %}
 
 Review the following Mobile Services functionality documentation for context and set up before implementation at these links:
 
-* [Getting started with Mobile Services](https://marketing.adobe.com/resources/help/en_US/mobile/gs.html)
-* [Create and managing apps](https://marketing.adobe.com/resources/help/en_US/mobile/manage_apps.html)
-* [Acquisition and marketing links](https://marketing.adobe.com/resources/help/en_US/mobile/acquisition_main.html)
-* [Push and in-app messaging](https://marketing.adobe.com/resources/help/en_US/mobile/in_app_messaging.html)
+* [Getting started with Mobile Services](https://experienceleague.adobe.com/docs/mobile-services/using/get-started-ug/gs.html)
+* [Create and managing apps](https://experienceleague.adobe.com/docs/mobile-services/using/manage-apps-ug/manage-apps.html?lang=en)
+* [Acquisition and marketing links](https://experienceleague.adobe.com/docs/mobile-services/using/acquisition-main-ug/acquisition-main.html?lang=en)
+* [Push and in-app messaging](https://experienceleague.adobe.com/docs/mobile-services/using/messaging-ug/in-app-messaging.html?lang=en)
 
 {% hint style="warning" %}
 Postbacks created from the Mobile Services interface are **not** supported. Please use a Launch rule with the Mobile Core extension to create a postback. [Signal extension and Rules Engine integration](../../resources/user-guides/signal-extension-and-rules-engine-integration.md).
 {% endhint %}
 
 {% hint style="warning" %}
-Before you configure the Mobile Services extension, ensure that you previously created apps in [Mobile Services](https://mobilemarketing.adobe.com). To create an app in Mobile Services, see [Add a new App](https://marketing.adobe.com/resources/help/en_US/mobile/t_new_app.html).
+Before you configure the Mobile Services extension, ensure that you previously created apps in [Mobile Services](https://mobilemarketing.adobe.com). To learn how to create an app in Mobile Services, see the [add a new App](https://experienceleague.adobe.com/docs/mobile-services/using/manage-apps-ug/t-new-app.html?lang=en) tutorial.
 {% endhint %}
 
 {% hint style="info" %}
-Looking for location functionality from Mobile Services? Try out Places Service, our point of interest management service. For more information, see [Places Service](https://docs.adobe.com/content/help/en/places/using/home.html).
+To use location functionality for Mobile Services, see the documentation on the [Places Service](https://experienceleague.adobe.com/docs/places/using/home.html?lang=en), the point of interest management service for Mobile Services.
```

Comment:
The `Location` functionality (https://experienceleague.adobe.com/docs/mobile-services/using/location-ug/location-overview.html?lang=en) is no longer supported by the `Mobile Services` extension. The `Places Service` is an replacement for the users interested in location-based services.